packages/dendron-next-server/pages/workspace/config.tsx

```diff
@@ -0,0 +1,326 @@
+                        <FormLabel margin={0}>Copy assets?</FormLabel>
+
+                        <Field name="site.copyAssets">
+                          {({
+                            form: { setFieldValue },
+                            field: { name, value },
+                          }) => (
+                            <Switch
+                              id={name}
+                              name={name}
+                              value={value}
+                              isChecked={values?.site?.copyAssets}
+                              onChange={(e) =>
+                                setFieldValue(name, e.target.checked)
+                              }
+                              colorScheme="positive"
+                            />
+                          )}
+                        </Field>
+                      </Stack>
+
+                      <FormHelperText>
+                        If enabled, assets will be copied from the vault to the
+                        site.
+                      </FormHelperText>
+                    </FormControl>
+                  </Stack>
+                </Box>
+
+                <Box as="fieldset">
+                  <Heading size="md" as="legend">
+                    Site hierarchies
+                  </Heading>
+
+                  <Stack spacing={4}>
+                    {configData.site?.siteHierarchies?.map(
+                      (hierarchy, index) => (
+                        <Tooltip
+                          label="Site hierarchies are not yet editable via this form."
+                          placement="top"
+                          hasArrow
+                        >
+                          <FormControl as={Stack} direction="row" spacing={2}>
+                            <Text
+                              htmlFor={`site.siteHierarchies[${index}]`}
+                              whiteSpace="nowrap"
+                              paddingTop={2} // align with text field
+                            >
+                              Hierarchy {index}
+                            </Text>
+
+                            <Box flexGrow={1}>
+                              <InputControl
+                                name={`site.siteHierarchies[${index}]`}
+                                placeholder="dendron"
+                                help="A hierarchy to publish"
+                                disabled
+                              />
+                            </Box>
+                          </FormControl>
+                        </Tooltip>
+                      )
+                    )}
+                  </Stack>
+                </Box>
+
+                <Stack direction="row" spacing={2} justify="end">
+                  <Button
+                    type="button"
+                    variant="ghost"
+                    onClick={() => resetForm()}
+                  >
+                    Reset
```

Comment:
notice that on reset, everything goes to "false/empty". Can we reset to the default instead?

Reply:
Hmm, i was intending `reset` to revert the form to the last-saved-config (a way of clearing your changes) -- as such, i've tweaked the copy of the button to `Clear changes` (:heavy_check_mark: @ 6bdc25e1). I can, additionally, add a "reset to defaults" button if you'd like (if yes, also please indicate whether it should immediately save the default values, or if the form should simply be populated with defaults)

tangentially-related commit that updated some copy/colors/behavior of the buttons too: d5e72a5c